src/NzbDrone.Core/Indexers/Definitions/GreatPosterWall.cs

```diff
@@ -248,171 +248,171 @@ public class GreatPosterWallResponse
 
 public class Response
 {
-        [JsonProperty("currentPage")]
-        public int CurrentPage { get; set; }
+    [JsonProperty("currentPage")]
+    public int CurrentPage { get; set; }
 
-        [JsonProperty("pages")]
-        public int Pages { get; set; }
+    [JsonProperty("pages")]
+    public int Pages { get; set; }
 
-        [JsonProperty("results")]
-        public List<Result> Results { get; set; }
+    [JsonProperty("results")]
+    public List<Result> Results { get; set; }
 }
 
 public class Result
 {
-        [JsonProperty("groupId")]
-        public int GroupId { get; set; }
+    [JsonProperty("groupId")]
```

Comment:
Really we can probably get rid of the `JsonProperty` attributes on all of these. They should work by default given they are just CamelCase